### PR TITLE
expect: fix Tcl tools

### PR DESCRIPTION
**Pull request by abbradar**
Fixes small tools bundled with `expect`, like `unbuffer`.

### CODE REVIEW

File: pkgs/tools/misc/expect/default.nix [hunk 1]
```diff
@@ -30,11 +31,16 @@ stdenv.mkDerivation {
...
     "--with-tcl=${tcl}/lib"
     "--with-tclinclude=${tcl}/include"
-    "--exec-prefix=$out"
+    "--exec-prefix=$(out)"
   ];
 
-  postInstall = let libSuff = if stdenv.isDarwin then "dylib" else "so";
-    in "cp expect $out/bin; mkdir -p $out/lib; cp *.${libSuff} $out/lib";
+  postInstall = let libSuff = if stdenv.isDarwin then "dylib" else "so"; in ''
+    for i in $out/bin/*; do
```
Comment (jaens) [inline pkgs/tools/misc/expect/default.nix]: This doesn't copy the libs any more? or is libSuff unneeded now?

Comment (abbradar) [inline pkgs/tools/misc/expect/default.nix]: Oh, you're right. I've fixed `make install`, so the copying should be unneeded now.

**Comment by abbradar** [comment]
Please wait a bit before merging; I want to be extra sure that `make install` is doing everything right.

**Comment by abbradar** [comment]
Yes, all good.

**Comment by lethalman** [comment]
Built locally on linux. Thanks.

**Pull request merged**